javav2/usecases/creating_fsa_app/src/main/java/com/example/fsa/handlers/SentimentHandler.java

```diff
@@ -0,0 +1,39 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.example.fsa.handlers;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.example.fsa.services.DetectSentimentService;
+import org.json.simple.JSONObject;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class SentimentHandler implements RequestHandler<Map<String, Object>, JSONObject> {
```

Comment:
Why not just have the class name reflect the Lambda handler definitions in the spec?

Reply:
Leave this the same - just a name